packages/app/src/api.ts

```diff
@@ -92,6 +92,23 @@ const api = {
         }),
     });
   },
+  useSilenceAlert() {
+    return useMutation<any, Error, { alertId: string; mutedUntil: string }>({
```

Comment:
Would it make sense for this to be a Date?
```suggestion
    return useMutation<any, Error, { alertId: string; mutedUntil: Date }>({
```